scripts/copy-files.js

```diff
@@ -15,6 +16,16 @@ function copyFile(file) {
   }).then(() => console.log(`Copied ${file} to ${buildPath}`));
 }
 
+function copyTypings(folder) {
```

Comment:
Just putting the `.d.ts` inside the `src` folder didn't copy it. I could also do

```dif
- const files = ['README.md', 'CHANGELOG.md', 'LICENSE'];
+ const files = ['README.md', 'CHANGELOG.md', 'LICENSE', 'typings/index.d.ts'];
```

That should work too. Would you prefer that?